src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see
+ * an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
+    .split(';');
+
+  return splittedString
+    .reduce((acc, word) => {
```

Comment:
Better would be to combine it into one chain, as the previous mentor mentioned

<img width="1015" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/30e6abb2-63ee-4f73-8e73-70a09a47fd35">